src/status_im/ui/screens/discover/views.cljs

```diff
@@ -85,7 +87,8 @@
                                              :current-account current-account}])]
          [empty-section :empty-hashtags :t/no-hashtags-discovered-title :t/no-hashtags-discovered-body])])))
 
-(defn recent-statuses-preview [current-account discoveries]
+(defn recent-statuses-preview [{:keys [current-account contacts discoveries]}]
+  (log/debug :papaya contacts)
```

Comment:
Remove

Reply:
Good catch! And now you know my grep keyword, how embarrassing :P

Reply:
that's my nightmare, i mean if once i will push all these weird keywords which i'm using for debugging :D